streaming/index.js

```diff
@@ -61,12 +63,12 @@ const parseJSON = (json, req) => {
      */
     if (req) {
       if (req.accountId) {
-        log.warn(req.requestId, `Error parsing message from user ${req.accountId}: ${err}`);
+        req.log.error({ err }, `Error parsing message from user ${req.accountId}`);
       } else {
-        log.silly(req.requestId, `Error parsing message from ${req.remoteAddress}: ${err}`);
+        req.log.error({ err }, `Error parsing message from ${req.remoteAddress}`);
       }
     } else {
-      log.warn(`Error parsing message from redis: ${err}`);
+      logger.error({ err }, `Error parsing message from redis`);
     }
     return null;
   }
```

Comment:
I'd ideally like to remove this mess, and instead handle the `parseJSON` method as a Result object `{ error, value }`, which would allow us to pass the error back to the call-site for logging, rather than performing the logging in this method.

I'm unsure if I should make this change now or not.